lookout/style/typos/utils.py

```diff
@@ -20,42 +21,50 @@ def read_frequencies(file: str) -> Dict[str, int]:
     """
     Read token frequencies from the file.
 
-    :param file: Path to the file containing tokens with frequencies.
-                 Every line has format "token count".
+    :param file: Path to the file containing space-separated word-frequency pairs one-per-line. \
+                 File should be in .csv or compressed .csv.xz format.
     :return: Dictionary of tokens frequencies.
     """
     frequencies = {}
-    with lzma.open(file, "rt") as f:
-        for line in f:
-            split = line.split()
-            frequencies[split[0]] = int(split[1])
+    try:
+        f = lzma.open(file, "rt")
+    except lzma.LZMAError:
+        f = open(file, "r")
+    for line in f:
+        split = line.split()
+        frequencies[split[0]] = int(split[1])
+    f.close()
```

Comment:
you should always close the file even if exception raises after you open the file. Please use the file extension to check file format 
```python
    open_ = lzma.open if file.endswith(".xz") else open
    with open_(file, "rt"):
        ...
```

and why `"rt"` and not `"rb"`?

Reply:
This `lzma` thing will be removed complitely in future PRs, can we just skip it here?

Reply:
Both `'r'` and `'rb'` work there actually, I think `'rt'` just stuck somehow.

Reply:
So should I change these chunks, considering that they will be removed in the future?